src/services/impressions/dto.js

```diff
@@ -29,7 +29,10 @@ module.exports = {
           return {
```

Comment:
We are doing all this transformation only because `key != keyName`, could we do something better?